radis/api/dbmanager.py

```diff
@@ -41,7 +41,7 @@
 import pandas as pd
 from dateutil.parser import parse as parse_date
 from joblib import Parallel, delayed
-from numpy import DataSource
+from numpy.lib.npyio import DataSource
```

Comment:
Seems it's only after numpy>=2.0.0 ?  https://numpy.org/doc/1.26/reference/generated/numpy.DataSource.html#numpy-datasource

To be sure, you can also do a try/except

```suggestion
try:
    from numpy.lib.npyio import DataSource
except ImportError:  # numpy <2.0.0?
    from numpy import DataSource
```

Reply:
yes indeed, it's only after numpy 2.0